app/models/lunch.rb

```diff
@@ -60,9 +68,26 @@ def chalmrest
           summary, price = entry.summary.split('@')
           summary = summary.strip
           price = price.strip
-
+          if restaurant[:name] == "Express" || restaurant[:name] == "Kårrestaurangen"
```

Comment:
Does it really need the check? Just do it for every restaurant 👍  (caching and so on...)
In the improbable case that other restaurants will post allergens as well 😄